<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>centerDialog</class>
 <widget class="QDialog" name="centerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>789</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>1061</width>
     <height>731</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="pageLayout" stretch="1,1">
    <item>
     <layout class="QVBoxLayout" name="pageLayout2" stretch="0,0,0,0,1,0,0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,0">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Camera:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboCameras"/>
        </item>
        <item>
         <widget class="QPushButton" name="buttonCamera">
          <property name="text">
           <string>Connect</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonCameraFind">
          <property name="text">
           <string>Find</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonCalibrateRt">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Calibrate Rt</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Resolution:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelCamProps">
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0,0,0">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Configuration:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="text">
           <string>D:/test/datei.xml</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>View</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Apply</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Mode:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="CameraWidget" name="cameraWidget" native="true">
        <property name="styleSheet">
         <string notr="true">border: 1px;
border-style: solid;
</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt;</string>
          </property>
          <property name="text">
           <string>Cursor Position [px]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelCursorPos">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt;</string>
          </property>
          <property name="text">
           <string>-, -</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt;</string>
          </property>
          <property name="text">
           <string>Point Position:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelLaserPoint">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt;</string>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0,0,0,0">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt;</string>
          </property>
          <property name="text">
           <string>Digitize from</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt;</string>
          </property>
          <property name="suffix">
           <string/>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>200.000000000000000</double>
          </property>
          <property name="value">
           <double>5.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt;</string>
          </property>
          <property name="text">
           <string>to</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
          <property name="styleSheet">
           <string notr="true">font: 75 12pt;</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>200.000000000000000</double>
          </property>
          <property name="value">
           <double>120.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
      <item>
       <widget class="QWidget" name="heightmapWidget" native="true"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QPushButton" name="buttonDigitize">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Digitize</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonMayavi">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>3D</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Load Points</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_7">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save Points</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save Image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonHeightmapClear">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CameraWidget</class>
   <extends>QWidget</extends>
   <header>cameraWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
